services/event_service.go

```diff
@@ -134,7 +134,7 @@ func (e *EventService) CreateAppEvent(ctx context.Context, newMessage *models.Ev
 		taskName := convoy.EventProcessor.SetPrefix(g.Name)
 
 		if eventDelivery.Status != datastore.DiscardedEventStatus {
-			err = e.eventQueue.Write(ctx, taskName, eventDelivery, 1*time.Second)
+			err = e.eventQueue.Write(context.Background(), taskName, eventDelivery, 1*time.Second)
```

Comment:
Oh, did I miss this one in my other PR?

Reply:
oh seen.

Reply:
Please don't undo it, I might have missed them